.github/workflows/ci.yml

```diff
@@ -210,6 +210,11 @@ jobs:
         run: ./scripts/ci/ci_perform_kind_cluster_operation.sh start
       - name: "Deploy app to cluster"
         run: ./scripts/ci/ci_deploy_app_to_kubernetes.sh
+      - uses: actions/upload-artifact@v2
+        if: failure()
```

Comment:
One small comment. It would be worth while to remove the if here. Those log might be useful to compare and see success vs. failure. And it does not cost us to have them also on success.

Reply:
Sure can do.

Reply:
(There are some limits on artifacts,  5gb for the whole repo across all builds I think I read somewhere, but if we're only storing compressed text logs it's probably not even close to an issue)